### ISSUE TITLE
GBIF feedback - lots of "&" in scientific names

### ISSUE THREAD

**Issue by ManonGros** [plazi/community#48]
This is another issue reported by @dshorthouse on the GBIF feedback system: https://github.com/gbif/portal-feedback/issues/3045
Some records from https://doi.org/10.5852/ejt.2019.551 show suspicious scientific names such as: 
> Zischkaia baku Nakahara & Zacca & Dias & Dolibaina & Xiao & Espeland & Casagrande & Mielke & Lamas & Huertas & Kleckner & Willmott 2019

**Comment by mguidoti** [comment]
Hi @ManonGros, 

Thanks for reporting it here!

Yesterday I set up a recurrent task so our processing team will, once a week, check if there are more issues citing Plazi's key [here](https://github.com/gbif/portal-feedback/issues?q=is%3Aissue+is%3Aopen+7ce8aef0-9e92-11dc-8738-b8a03c50a862).

Also, I finally managed to tame the help desk workflow... This particular issue was assigned to @valdenar and it's likely that it will be solved in the upcoming hours.

We will post updates [here](https://github.com/gbif/portal-feedback/issues?q=is%3Aissue+is%3Aopen+7ce8aef0-9e92-11dc-8738-b8a03c50a862) as soon as we have these issues figured out.

Thanks!
